tests/lzma.rs

```diff
@@ -142,36 +216,36 @@ fn big_file() {
 fn decompress_empty_world() {
     #[cfg(feature = "enable_logging")]
     let _ = env_logger::try_init();
-    let mut x: &[u8] = b"\x5d\x00\x00\x80\x00\xff\xff\xff\xff\xff\xff\xff\xff\x00\x83\xff\
-                         \xfb\xff\xff\xc0\x00\x00\x00";
-    let mut decomp: Vec<u8> = Vec::new();
-    lzma_rs::lzma_decompress(&mut x, &mut decomp).unwrap();
-    assert_eq!(decomp, b"")
+    assert_decomp_eq(
+        b"\x5d\x00\x00\x80\x00\xff\xff\xff\xff\xff\xff\xff\xff\x00\x83\xff\
+                         \xfb\xff\xff\xc0\x00\x00\x00",
```

Comment:
Nit: indentation should better align here.

Reply:
The rust-analyzer vscode plugin displays inlay hints and this was offsetting the first line in my IDE. Thankfully this can be disabled :) .